Gemfile.lock

```diff
@@ -82,21 +82,21 @@ GEM
     ast (2.4.2)
     autoprefixer-rails (10.2.4.0)
       execjs
-    aws-eventstream (1.1.0)
-    aws-partitions (1.425.0)
-    aws-sdk-core (3.112.0)
+    aws-eventstream (1.1.1)
+    aws-partitions (1.445.0)
+    aws-sdk-core (3.114.0)
       aws-eventstream (~> 1, >= 1.0.2)
       aws-partitions (~> 1, >= 1.239.0)
       aws-sigv4 (~> 1.1)
       jmespath (~> 1.0)
-    aws-sdk-kms (1.42.0)
+    aws-sdk-kms (1.43.0)
       aws-sdk-core (~> 3, >= 3.112.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.88.0)
+    aws-sdk-s3 (1.93.1)
       aws-sdk-core (~> 3, >= 3.112.0)
       aws-sdk-kms (~> 1)
       aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.2.2)
+    aws-sigv4 (1.2.3)
```

Comment:
### [aws-sigv4](https://github.com/aws/aws-sdk-ruby)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.2.2 → 1.2.3

[_[change-log](https://github.com/aws/aws-sdk-ruby/tree/master/gems/aws-sigv4/CHANGELOG.md), [source-code](https://github.com/aws/aws-sdk-ruby/tree/master/gems/aws-sigv4)_]





<details>
<summary>Commits</summary>

A change of **4** commits. See the full changes on [the compare page](https://github.com/aws/aws-sdk-ruby/compare/1.2.2...1.2.3).

These are the individual commits:
- (9e98d9c) [Added documentation for part_number option of MultipartUploa…](https://github.com/aws/aws-sdk-ruby/commit/9e98d9c4a86956877a71eb9d2338e4c9121876cd)
- (c182b0a) [Fetch console output for instance.](https://github.com/aws/aws-sdk-ruby/commit/c182b0afda46124aea7d8899261826a331427786)
- (22bc7e8) [Merge commit '9e98d9c4a86956877a71eb9d2338e4c9121876cd'](https://github.com/aws/aws-sdk-ruby/commit/22bc7e8209824c73e61658dad8b10166a59c962f)
- (ddbd83f) [Version 1.2.3 of the AWS SDK for Ruby](https://github.com/aws/aws-sdk-ruby/commit/ddbd83fa5acb83d328e6fdeb2ccd75e1affdd2ee)


</details>